src/test/scala/com/campudus/tableaux/TableauxTestBase.scala

```diff
@@ -20,19 +21,25 @@ case class TestCustomException(message: String, id: String, statusCode: Int) ext
 trait TableauxTestBase extends TestVerticle {
   val DEFAULT_PORT = 8181
 
+  lazy val config: JsonObject = jsonFromFile("../conf.json")
+  lazy val databaseAddress: String = config.getObject("database", Json.obj()).getString("address", "campudus.asyncdb")
+
+  private def readJsonFile(f: String): String = Source.fromFile(s"$f").getLines().mkString
```

Comment:
1. `config` is used not only in `asyncBefore()`
2. IDEA tells me `Empty-paren method accessed as parameterless` if I remove the parentheses. Problem is, that the implementation uses parentheses! 
3. see 1.1.